### DISCUSSION TITLE
Pivpn not working with asus router

### DISCUSSION THREAD

**Discussion by Icyfiremario** [pivpn/pivpn#1679] [Support] [answered]
I recently replaced the router that came with my internet with an Asus RT-AC68U and I'm having trouble getting anything to connect to Pivpn. I have the port open and my certificates have not expired so I do not understand how this is happening.

**Comment by coolapso** [comment]
Is your router directly connection you to the internet?

ISPs lease  the IP addresses based on the MAC of the router. If you change router your Public IP address changes as well. Double check if the IP address in the configuration matches the IP address you currently have.

**Comment by Icyfiremario** [comment]
> Is your router directly connection you to the internet?
> 
> 
> 
> ISPs lease  the IP addresses based on the MAC of the router. If you change router your Public IP address changes as well. Double check if the IP address in the configuration matches the IP address you currently have. 

No, I have a separate modem so it was just a matter of plugging in the router and setting it up

**Comment by coolapso** [comment]
https://docs.pivpn.io/faq/#how-do-i-troubleshoot-connection-issues

Moving to discussions.

**Comment by Icyfiremario** [comment, marked as answer]
I just had reconfigure my vpn server cus my IP actually changes now
